src/editable-heading/editable-heading.tsx

```diff
@@ -24,6 +26,7 @@ export type EditableHeadingProps = Omit<
   level?: Levels;
   headingClassName?: string | null;
   inputClassName?: string | null;
+  textAreaWrapperClassName?: string | null;
```

Comment:
We already have the ability to pass `Size.FULL` to the `size` prop, but it stopped working for multiline cases after adding textarea. So we need to solve this within the component itself, rather than forcing the client to pass `className` with `width: 100%`. For example, we can apply a style with `width: 100%` to `.textAreaWrapper` if `size` is `Size.FULL`.

Reply:
done

Reply:
`textAreaWrapperClassName` is now unused and should be removed.

Reply:
done